radixdlt/src/main/java/com/radixdlt/consensus/Round.java

```diff
@@ -26,6 +26,10 @@ public Round next() {
 		return new Round(round + 1);
 	}
 
+	public long number() {
```

Comment:
For serialization.

Reply:
number() is a non-ideal name. How about just round()?

Reply:
Mmmm.  I thought we would end up with code like "round.round()" which doesn't help much when reading code.  Thought "round.number()" was better.  Can change it if you strongly object?

Reply:
I'm going to leave this as is for now.